packages/components/Atoms/Icon/Icon.vue

```diff
@@ -1,12 +1,24 @@
+<!-- credits to https://github.com/Justineo/vue-awesome/blob/master/src/components/Icon.vue -->
 <template>
-  <svg v-if="validSymbol" :class="iconClass" :style="iconStyle" aria-hidden="true" overflow="visible">
-    <!-- Use both xlink:href and href for browser support https://css-tricks.com/on-xlinkhref-being-deprecated-in-svg/ -->
-    <use :xlink:href="'#' + iconPrefix + symbol" :href="'#' + iconPrefix + symbol"></use>
+  <svg v-if="icon" :class="iconClass" :style="iconStyle" :viewBox="box" aria-hidden="true" overflow="visible">
+    <template v-if="iconPropsSVG">
```

Comment:
this is to support the svg loader. In this case the loader will only load the clients svg as I have commented out https://github.com/dpc-sdp/ripple/pull/774/files#diff-fd785211170dde832ab79dde7d68ce78L35